tests/recipes/cad1/task2/baseline/test_audio_manager.py

```diff
@@ -32,3 +32,18 @@ def test_save_audios(tmp_path):
     assert sample_rate == pytest.approx(
         audio_manager.sample_rate, rel=pytest.rel_tolerance, abs=pytest.abs_tolerance
     )
+
+
+@pytest.mark.skip(reason="Not implemented yet")
+def test_clip_audio():
+    """Test clip_audio method."""
+
+
+@pytest.mark.skip(reason="Not implemented yet")
+def test_get_lufs_level():
+    """Test get_lufs_level method."""
+
+
+@pytest.mark.skip(reason="Not implemented yet")
+def test_scale_to_lufs():
+    """Test scale_to_lufs method."""
```

Comment:
More homework

Reply:
Yes, but perhaps not urgent and I doesn't necessarily need a test for each function. For some of the CEC, CPC recipes I felt it was sufficient to test the top level run function that then covers the low level functions. This will still pick up a breaking change after which it wouldn't be hard to isolate the function where it happens.